.github/workflows/ci_workflows.yml

```diff
@@ -0,0 +1,124 @@
+name: CI
+
+# TODO: cron on a separate YAML?
+on:
+  push:
+  pull_request:
+
+# The following is needed to avoid issues if e.g. Matplotlib tries
+# to open a GUI window.
+env:
+  SETUP_XVFB: True
```

Comment:
I think this was a ci-helpers thing

Reply:
Yeah, this PR been open for a while and probably need upgrades.